src/core_modules/capture-core/components/WidgetEventSchedule/ScheduleDate/ScheduleDate.component.js

```diff
@@ -27,7 +27,7 @@ const ScheduleDatePlain = ({
 }: Props) => (<>
     <div className={classes.container}>
         <DateField
-            value={scheduleDate}
+            value={moment(scheduleDate).format('YYYY-MM-DD')}
```

Comment:
hey @JoakimSM, when I was testing it I sometime got the schedule date with time (i.e: 2021-09-01T12:00) but probably because of not converting. It should be fixed now by convertToForm so I will revert this change